src/api/types/generated-schema.ts

```diff
@@ -8602,6 +8652,22 @@ export interface operations {
       };
     };
   };
+  /** Validates the definition of a new planting site. */
+  validatePlantingSite: {
```

Comment:
Is this supposed to be in here?

Reply:
I re-generated the types and just pulled in all of the changes, but I'll remove the unrelated changes to keep the PR clean.